MTA-6811, 6812, 6813, 6814, 6815, 6816 CQA April Developer Lightspeed work (#357)
migtools/mta-documentation: docs/topics/developer-lightspeed/proc_configuring-llm-podman-desktop.adoc

= Configuring the LLM in Podman Desktop

[role="_abstract"]

> Use the Podman AI lab extension to select an open source model from a curated list of models and run it locally in your system. 

Reviewer: Why before what:
```suggestion
To select an open source model from a curated list of models and run it locally in your system, use the Podman AI lab extension. 
```

> The code fix suggestions that a model generates depend on the model's capabilities. Models from the Podman AI Lab are insufficient for the complexity of code changes needed to fix issues that {ProductShortName} discovers. You must not use such models in a production environment.

Reviewer: ```suggestion
The code fix suggestions that a model generates depend on the model's capabilities. Models from the Podman AI Lab are insufficient for the complexity of code changes needed to fix issues that {ProductShortName} discovers. Do not use such models in a production environment.
```
